slsDetectorGui/src/qTabMeasurement.cpp

```diff
@@ -213,23 +213,16 @@ void qTabMeasurement::SetupTimingMode() {
             index[i] = model->index(i, comboTimingMode->modelColumn(),
                                     comboTimingMode->rootModelIndex());
             item[i] = model->itemFromIndex(index[i]);
+            item[i]->setEnabled(false);
         }
-
-        item[(int)GATED]->setEnabled(false);
-        item[(int)BURST_TRIGGER]->setEnabled(false);
-        item[(int)TRIGGER_GATED]->setEnabled(false);
-        switch (det->getDetectorType().squash()) {
-        case slsDetectorDefs::EIGER:
-            item[(int)GATED]->setEnabled(true);
-            item[(int)BURST_TRIGGER]->setEnabled(true);
-            break;
-        case slsDetectorDefs::MYTHEN3:
-            item[(int)GATED]->setEnabled(true);
-            item[(int)TRIGGER_GATED]->setEnabled(true);
-            break;
-        default:
-            break;
+        try {
```

Comment:
Nice!

Reply:
Need to do the same for getdaclist etc, but this might not be so easy. added it to backlog